js/functions.js

```diff
@@ -0,0 +1,30 @@
+// Функция для проверки длины строки
+
+let lessOrEqual = (string, length) => string.length <= length;
+console.log(lessOrEqual("бубубу", 12));
+
+// Функция для проверки, является ли строка палиндромом
+
+// let isPalindrome = () => {
+// 	let invertedString = '';
+// 	for (let i = string.length - 1; i >= 0; i--) {
+// 		invertedString += string[i];
+// 	}
+// 	return string === invertedString;
+// }
+
+// or
+
+let isPalindrome = (string) => {
+	string = string.replaceAll(" ", "").toLowerCase();
+	let reversed = string.split("").reverse().join("");
+	return reversed === string;
+}
+
+console.log(isPalindrome("ДОвод"));
+
+// функция, преобразующая строку в цифры
+
+let getNumber = (string) =>  typeof(string) === "number" ? String(string) : parseInt(string.replace(/[^0-9]/g,""));
```

Comment:
Не все тестовые кейсы из задания проходит
![image](https://github.com/htmlacademy-javascript/2070743-kekstagram-29/assets/47776594/217a9ad5-bf53-49b6-bfcd-d4f335aad144)
Можно так поправить

```suggestion
const getNumber = (string) => {
	string = string.toString();
	return parseInt(string.replace(/[^0-9]/g,""), 10);
}
```